codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/SyncClientClass.java

```diff
@@ -418,7 +421,7 @@ protected MethodSpec.Builder waiterOperationBody(MethodSpec.Builder builder) {
                                     poetExtensions.getSyncWaiterInterface());
     }
 
-    protected static MethodSpec updateSdkClientConfigurationMethod(
+    protected MethodSpec updateSdkClientConfigurationMethod(
```

Comment:
This is to enable the method to access the model instance. I am unsure as to why a code gen method is static in the first place.